ccmlib/cluster.py

```diff
@@ -470,23 +481,27 @@ def show(self, verbose):
             else:
                 node.show(only_status=True)
 
-    def start(self, no_wait=False, verbose=False, wait_for_binary_proto=False, wait_other_notice=False, jvm_args=None, profile_options=None, quiet_start=False):
+    def start(self, no_wait=False, verbose=False, wait_for_binary_proto=False, wait_other_notice=False, jvm_args=None,
+              profile_options=None, quiet_start=False, parallel_start=None):
         if jvm_args is None:
             jvm_args = []
+        if parallel_start is None:
+            parallel_start = self.parallel_start_supported
```

Comment:
this should be explicit, not like that.

this gonna break all of dtest

Reply:
why ? we already start scylla nodes in parallel